strategy-cache-only/non-controlled.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Cache only: non controlled page - ServiceWorker Cookbook</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+   body {
+     text-align: center;
+   }
+  </style>
+</head>
+<body>
+  <h1>Always synchronized</h1>
```

Comment:
I'm not sure 'synchronized' works here, perhaps 'always fresh from the server' would be more clear. Synchronized makes me think of synchronising content both ways and that's not what's happening...